### DISCUSSION TITLE
Is this available for commericial use?

### DISCUSSION THREAD

**Discussion by patrickphat** [givanz/VvvebJs#381] [Q&A] [answered]
Hi Givanz,

Thanks a lot for building VvvebJS, it's amazing. I just wonder what makes you spend time on this because it seems freely offered?

Is it ok if I commercialize it? I will give you a share profit :)

**Comment by givanz** [comment, marked as answer]
Hi 

Thank you.

VvvebJs is primarly used in Vvveb CMS a full fledged platform that allows anyone to easily build professional/premium websites without the need of paid plugins.

VvvebJs is licensed under Apache 2.0 you can freely use it in commercial projects, you only need to keep the copyright notice in the source code.
